libs/sqlite3/Makefile

```diff
@@ -1,24 +1,22 @@
+# SPDX-License-Identifier: GPL-2.0-only
 #
-# Copyright (C) 2006-2014 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
+# Copyright (C) 2006-2025 OpenWrt.org
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=sqlite
 PKG_VERSION:=3.51.0
 PKG_SRC_VERSION:=3510000
-PKG_RELEASE:=1
+PKG_RELEASE:=2
+PKG_ABI_VERSION:=0
```

Comment:
I am just thinking about it loud as I havent look at it further. Usually what is is done in the other packages is that there is no ``PKG_ABI_VERSION``, right now, it seems extra to me. We might keep the previous ``ABI_VERSION:=0``, but change the install section to this:

```
$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsqlite3.so.$(ABI_VERSION)* $(1)/usr/lib
```

Not tested actually... maybe based on your changes it should be like this:
```
$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsqlite3.so.{$(PKG_VERSION),$(ABI_VERSION)} $(1)/usr/lib
```